bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -417,3 +418,19 @@ def pythonize_rdataframe(klass):
                                       HistoProfileWrapper,
                                       model_class)
         setattr(klass, method_name, getter)
+
+
+@pythonization("RInterface<", ns="ROOT::RDF", is_prefix=True)
```

Comment:
Is there any reason why you kept a separate pythonizor function for `Filter`, instead of just adding these lines to `pythonize_rdataframe`? Having just one pythonizor for RDF is better performance-wise, because there's less comparisons involved.